tests/reading-tests.md

```diff
@@ -0,0 +1,397 @@
+
+<!-- prettier-ignore-start -->
+```js
+() => {
+    // ...
+}
+```
+<!-- prettier-ignore-end -->
+
+Let's approach the contents of this function piece-by-piece.
+
+### Hypothesizing About `test`
+
+Inside the anonymous function started in `describe`, there is a block of code that begins with `test()`.
+
+<!-- prettier-ignore-start -->
+```js
+  test('returns Fizz on multiples of 3', () => {
+    // ...
+  });
+```
+<!-- prettier-ignore-end -->
+
+This is also a function being invoked. This `test` function seems to take in two arguments, the string literal `'returns Fizz on multiples of 3'` and an anonymous arrow function.
+
+Leila notices that this pattern feels very similar to the `describe` block. 🤔 They should keep that observation, because it's a good one!
+
+### Arrange
+
+Inside the next anonymous arrow function, we see the following code:
+
+<!-- prettier-ignore-start -->
+```js
+    // Arrange
+    const num = 9;
+```
+<!-- prettier-ignore-end -->
+
+It's a comment indicating an "Arrange" step and a `const` variable `num` assigned to the number `9`.
+
+Leila is an overthinker, and they could dig deeper into what this line means. However, this time, they will choose to make only one theory: the variable `num` will be used later in the test.
+
+### Act
+
+We can move on to reading the next set of lines:
+
+<!-- prettier-ignore-start -->
+```js
+    // Act
+    const result = fizzBuzz(num);
+```
+<!-- prettier-ignore-end -->
+
+Leila recognizes this as the "Act" step in a test, where the behavior we're testing (a `fizzBuzz` function) should be "acted on," or invoked.
+
+Here, we are invoking `fizzBuzz`, passing in `num`, and saving the return value into a `const` variable named `result`.
+
+### Assert
+
+Let's dive into the final line, which features some more unfamiliar syntax.
+
+<!-- prettier-ignore-start -->
+```js
+    // Assert
+    expect(result).toEqual('Fizz');
+```
+<!-- prettier-ignore-end -->
+
+Leila knows that the assert step is the step that determines whether a test passes or fails.
+
+They've seen the `test` block include the string "returns Fizz on multiples of 3," which Leila suspects is relevant to the Assert step of this test.
+
+Reading the line from left-to-right, it seems like `expect(result)` is also a function call. The function's name is `expect` and we're passing in `result`.
+
+However, the line doesn't stop there. Leila sees `.toEqual('Fizz')`. Recalling other JavaScript syntax, Leila suspects that this is a function call, where the function is named `toEqual` and the argument is `'Fizz'`.
+
+It looks like this function call operates from an object. That object must be `expect(result)`.
+
+### Reading it Out Loud
+
+Leila can now take a step back and read this line overall. They notice a sentence that the code forms: "Expect the value of `result` to equal `'Fizz'`."
+
+The test will pass if `result`'s value is `'Fizz'`, and fail if it isn't.
+
```

Comment:
suggest a callback to provide additional perspective on why it's okay to make all these assumptions: code is written by people for people and we want it to make sense!
```suggestion

### !callout-info

## Is It Safe to Jump to this Conclusion?

It may seem reckless to try reading this code as a sentence rather than code. After all, JavaScript doesn't know what any of these words mean, and it definitely doesn't know what a sentence is!

<br />

But we should remember that this test library was written by _people_ for other _people_ to use. They want their code to be understandable, and so they wrote it so that it makes intuitive sense, even before we have a deep technical understanding. We should attempt to write our own code in such an understandable style too!

### !end-callout

```

Reply:
this is brilliant and i love it a lot, thank you!!